### ISSUE TITLE
mat-datepicker: Screen Reader doesn't announce the week day while navigating through the days in calender

### ISSUE THREAD

**Issue by vmyakala** [angular/components#22711]
Screen reader doesn't announce the week day while navigating through the days in calendar. Current behavior, it announces "May 21, 2021" but the user doesn't know which weekday is it. 

It will be helpful if the screen readers says, "Friday, May 21 - 2021" something like this.

**Comment by mmalerba** [comment]
Closing as dup of #21357
